Kuzzle.Tests/API/Controllers/BulkControllerTest.cs

```diff
@@ -0,0 +1,115 @@
+﻿using KuzzleSdk.API.Controllers;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Kuzzle.Tests.API.Controllers {
+  public class BulkControllerTest {
+    private readonly BulkController _bulkController;
+    private readonly KuzzleApiMock _api;
+
+    public BulkControllerTest() {
+      _api = new KuzzleApiMock();
+      _bulkController = new BulkController(_api.MockedObject);
+    }
+
+    [Fact]
+    public async void ImportAsyncTestSuccess() {
+
+      JObject expected = JObject.Parse(@"{items: [{create: {_id: 'documentId', status: 201}}]}");
+
+      _api.SetResult(JObject.Parse(@"{result: {items: [{create: {_id: 'documentId', status: 201}}]}}"));
+
+      JObject result = await _bulkController.ImportAsync(
+        "foo",
+        "bar",
+        JArray.Parse(@"[]")
+        );
+
+      _api.Verify(new JObject {
+        {"index", "foo"},
+        {"collection", "bar"},
+        {"controller", "bulk"},
+        {"action", "import"},
+        {"body", new JObject {
+          {"bulkData", JArray.Parse(@"[]")}
+        } }
+      });
+
+      Assert.Equal<JObject>(expected, result, new JTokenEqualityComparer());
+    }
+
+    [Theory]
+    [InlineData(false)]
+    [InlineData(true)]
+    public async void mWriteAsyncTestSuccess(bool notify) {
```

Comment:
Public methods should be capitalized.

```suggestion
    public async void MWriteAsyncTestSuccess(bool notify) {
```

Reply:
At the time, I didn't know what to do with this special case.

I've changed it for a Capital letter.